src/libsync/propagateremotemove.cpp

```diff
@@ -51,10 +51,18 @@ void MoveJob::start()
     for (auto it = _extraHeaders.constBegin(); it != _extraHeaders.constEnd(); ++it) {
         req.setRawHeader(it.key(), it.value());
     }
+
+    QUrl urlForLog;
     if (_url.isValid()) {
-        sendRequest("MOVE", _url, req);
+        urlForLog = _url;
+        sendRequest("MOVE", urlForLog, req);
```

Comment:
Can you factor the if such that `urlForLog` becomes const and is initialized via a ternary operator ?
That would remove the need for an if